app/models/user.rb

```diff
@@ -366,20 +366,14 @@ def identity_not_verified?
     !identity_verified?
   end
 
-  def identity_verified?(service_provider: nil)
-    active_profile.present? && !reproof_for_irs?(service_provider: service_provider)
+  def identity_verified?
+    active_profile.present?
   end
 
   def identity_verified_with_biometric_comparison?
     BIOMETRIC_COMPARISON_IDV_LEVELS.include?(active_profile&.idv_level)
   end
 
-  def reproof_for_irs?(service_provider:)
```

Comment:
I feel like `reproof_for_irs` is not exactly the Attempts API, and could maybe be removed in a separate PR? But I still think it should be removed, regardless

Reply:
Yeah I started weighing this as soon as I realized how deep the rabbit hole went for `reproof_for_irs`. The main reason for including it here is that the logic of this method is entirely based around `irs_attempts_api_enabled`. But I think it could be something to split off to a separate pull request to simplify some of the changes / reduce risk if we wanted, and in retrospect I probably would have done that if I had known ahead of time just how much `reproof_for_irs` stuff there was.